setup.py

```diff
@@ -43,9 +44,10 @@ def is_requirement(line):
     """
     return line and not line.startswith(('-r', '#', '-e', 'git+', '-c'))
 
+
 setup(
     name='xblock-sql-grader',
-    version='0.0.1',
+    version='0.0.2',
```

Comment:
I think we could at least bump the minor version on this; adding `codejail` integration for grading seems worth at least bumping that, if not even the major version.